app/controllers/content_items_controller.rb

```diff
@@ -14,6 +15,8 @@ class ContentItemsController < ApplicationController
   def show
     load_content_item
 
+    load_taxonomy_navigation if page_in_scope?
```

Comment:
Should we use the `show_new_navigation?` helper here?

Reply:
Yep, that's true